### PR TITLE
BLASR-2.1-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/58d840cc30b679248354 for a full test report.

File: easybuild/easyconfigs/b/BLASR/BLASR-2.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/BLASR/BLASR-2.1-goolf-1.4.10.eb]: minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!

**Comment by rjeschmi** [comment]
```
== temporary log file in case of crash /tmp/eb-ax50ei/easybuild-Jr03yV.log
ERROR: Failed to process easyconfig /tmp/eb-ax50ei/files_pr922/BLASR-2.1-goolf-1.4.10.eb: NO LONGER SUPPORTED since v2.0: Easyconfig parameter 'makeopts' is replaced by 'buildopts';
 see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html for more information
```

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/b/BLASR/BLASR-2.1-goolf-1.4.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/b/BLASR/BLASR-2.1-goolf-1.4.10.eb]: done, see below

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/aa9fc2bff90d65348022 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**